spec/features/incoming_mail_handler_spec.rb

```diff
@@ -0,0 +1,79 @@
+describe "Handles incoming email" do
+  let(:proposal) { FactoryGirl.create(:proposal, :with_parallel_approvers) }
```

Comment:
Perhaps all of us should sit down and discuss at some point. Yoz and I had a 10 minute convo about it. I find the arguments in this blog post to be persuasive: https://robots.thoughtbot.com/lets-not

Agree that many many people who write rspec use `let` statements so they are not confusing to new people. But I do prefer to have all of the dependencies for each test called explicitly in each test so I know what I'm dealing with.